firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -90,6 +92,8 @@ public InstallationResponse createFirebaseInstallation(
                   apiKey));
 
       HttpsURLConnection httpsURLConnection = (HttpsURLConnection) url.openConnection();
+      httpsURLConnection.setConnectTimeout(TIMEOUT_IN_MILLI_SECS);
+      httpsURLConnection.setReadTimeout(TIMEOUT_IN_MILLI_SECS);
```

Comment:
I want to verify if this is the right approach. I will then add it to all the network calls.